pkg/apis/kops/componentconfig.go

```diff
@@ -755,6 +755,26 @@ type CloudConfiguration struct {
 	Openstack *OpenstackConfiguration `json:"openstack,omitempty"`
 }
 
+// ClusterAutoscalerConfig determines the cluster autoscaler configuration
+type ClusterAutoscalerConfig struct {
+	// Expander determines the strategy for which instance group gets expanded
+	// Supported values: least-waste, most-pods, random
+	// Default: least-waste
```

Comment:
```suggestion
	// Expander determines the strategy for which instance group gets expanded.
	// Supported values: least-waste, most-pods, random.
	// Default: least-waste
```

Otherwise the generated docs look odd.